lib/modules/quarantineHide.js

```diff
@@ -0,0 +1,62 @@
+/* @flow */
+
+import { Module } from '../core/module';
+import { $ } from '../vendor';
+
+export const module: Module<*> = new Module('quarantineHide');
+
+module.moduleName = 'qhideName'; // i18n
+module.category = 'appearanceCategory';
+module.description = 'qhideDesc'; // i18n
+module.options = {
+	hideFlair: {
+		title: 'qhideFlairTitle', // i18n
+		type: 'boolean',
+		value: false,
+		description: 'qhideFlairDesc', // i18n
+	},
+	hideSidebar: {
+		title: 'qhideSidebarTitle', // i18n
+		type: 'boolean',
+		value: false,
+		description: 'qhideSidebarDesc', // i18n
+	},
+	fixHeaderImg: {
+		title: 'qhideFixHeaderImgTitle', // i18n
+		type: 'boolean',
+		value: false,
+		description: 'qhideFixHeaderImgDesc', // i18n
+	},
+	hideWarning: {
+		title: 'qhideWarningTitle', // i18n
+		type: 'boolean',
+		value: false,
+		description: 'qhideWarningDesc', // i18n
+	},
+};
+
+module.include = ['linklist'];
+
+module.go = () => {
+	if (module.options.hideFlair.value) {
+		removeFlair();
+		window.addEventListener('neverEndingLoad', removeFlair);
+	}
+	if (module.options.hideSidebar.value) {
+		const $notice = $('.quarantine-notice');
+		if (module.options.fixHeaderImg && $notice.length !== 0) {
```

Comment:
Could you put this `if fixHeaderImg` at the same level as `if hideSidebar`? Since these are unrelated chunks of functionality, they should be in separate blocks. 

It looks like you combined them to take advantage of `$notice.length` to determine "does this page show quarantined UI?"  It would be clearer to add something like `const inQuarantinedSubreddit = !!(document.querySelector('.quarantine-notice'))` to the top of `go()`. 
It would be even nicer to add `inQuarantinedSubreddit = _.once(() => { look for hints of quarantine });` to `lib/utils/location` (and import it into `lib/utils` for importing into modules.) You could then use that function in these `if` predicates.